.github/workflows/ci.yaml

```diff
@@ -60,16 +75,17 @@ jobs:
       MANIFEST_ARTIFACT_NAME: ${{ steps.manifest-branch.outputs.MANIFEST_ARTIFACT_NAME }}
       MANIFEST_BRANCH: ${{ steps.manifest-branch.outputs.MANIFEST_BRANCH }}
       MERGE_BUMPED_MANIFEST: ${{ steps.manifest-branch.outputs.MERGE_BUMBED_MANIFEST }}
+      TEST_SUBSET: ${{ steps.testset.outputs.TEST_SUBSET }}
     steps:
-      - name: Cancel workflow run if the trigger is a draft PR
-        id: cancel-if-draft
-        if: github.event_name == 'pull_request' && github.event.pull_request.draft == true
-        run: |
-          echo "Cancelling workflow for draft PR"
-          curl -X POST -H "Authorization: token ${{ github.token }}" \
-                -H "Accept: application/vnd.github.v3+json" \
-                "https://api.github.com/repos/${{ github.repository }}/actions/runs/${{ github.run_id }}/cancel"
-          while true; do sleep 1; done  # blocks execution in case workflow cancellation takes time
+      # - name: Cancel workflow run if the trigger is a draft PR
```

Comment:
Remove dead lines?

Reply:
shouldn't be dead in the first place

Reply:
You're going to have another PR for this step?